apps/web/components/booking/BookingDescription.tsx

```diff
@@ -55,7 +55,7 @@ const BookingDescription: FC<Props> = (props) => {
         }
       }
     }
-  }, []);
+  }, [setDuration, eventType.length, eventType.metadata?.multipleDuration, duration]);
```

Comment:
This works till I figure out why duration param is being removed.

Reply:
Using shallow:false on `useRouterQuery` also fixes the issue.

Reply:
so can we merge this?

Reply:
Yeah, it's safe to merge and also fixes the issue.

But the root cause for why it happens only in embed isn't clear.